devops/openshift/forms-deploy.yml

```diff
@@ -164,11 +164,11 @@ objects:
     spec:
       scaleTargetRef:
         apiVersion: apps/v1
-        kind: ReplicationController
-        name: ${NAME}-rc
+        kind: DeploymentConfig
+        name: ${NAME}
       minReplicas: "${{REPLICAS}}"
-      maxReplicas: 10
-      targetCPUUtilizationPercentage: 80
+      maxReplicas: 6
```

Comment:
just for my understanding my `minReplicas` is set as a variable and for `maxReplicas` we directly assigned the value

Reply:
Yes, right now I am not changing it but the REPLICAS is also set in the parameters as 2 right now.